images/ci-operator-minimal/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
+
+RUN microdnf install -y git python3
```

Comment:
Can you check if we can deprecate that? IIRC the git dependency exists to allow users to pass their own git revisions to test stuff. I thing we long passed this idea. Also, python3, why?

Reply:
I can't tell anything about git but python is required, see [this](https://redhat-internal.slack.com/archives/CBN38N3MW/p1717595757867709)